packages/components/Avatar/src/AvatarTokens.android.ts

```diff
@@ -0,0 +1,203 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { AvatarTokens } from '.';
+import { globalTokens } from '@fluentui-react-native/theme-tokens';
+
+export const defaultAvatarTokens: TokenSettings<AvatarTokens, Theme> = (t: Theme) =>
+  ({
```

Comment:
I'm realizing that platform specific token files are difficult to review - ex. it's hard for reviewers to know what the diff is between this file and the other AvatarTokens.ts file. Thinking in the future for my PRs if I make a new platform specific token file, I'll try to commit an exact duplicate first and then make the platform-specific changes after so there's a diff people can reference. 

Doing a brief manual diff on this - seems like mostly some colors changed, and we also added brandinverted/accent colors. Are there any other changes?

Reply:
That sounds about right in terms of changes to this file. And it's a good suggestion if we can try to commit unchanged version first, thank you!